src/pages/wishlist/WishlistCard.jsx

```diff
@@ -9,23 +18,36 @@ const WishlistCard = () => {
           alt="phone"
           className="card__img"
         />
-        <button className="card__remove-wishlist">
+        <button
+          className="card__remove-wishlist"
+          onClick={() =>
+            removeItemFromWishlist({
+              _id,
+              currentUser,
+              stateDispatch,
+              setIsLoading,
+            })
+          }
```

Comment:
I think you can use `useContext` in the your custom hook useAppActions, so you wont have the need to pass these arguments in the function

Reply:
Users can remove items from the wishlist. I didn't show this in the video. 
Will use Context in the hook, thanks for the suggestion.